rmgpy/data/kinetics/familyTest.py

```diff
@@ -639,6 +641,54 @@ def testSaveFamily(self):
             shutil.rmtree(os.path.join(settings['test_data.directory'], 'testing_database/kinetics/families/intra_H_copy'))
 
 
+###################################################
+class TestLoadFilterFits(unittest.TestCase):
```

Comment:
I know I said that you could keep it in familyTest.py, but I think this is probably more suited for kineticsTest.py since you're loading the KineticsDatabase functionality.